components/camel-tracing/src/main/java/org/apache/camel/tracing/TagConstants.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.tracing;
+
+public class TagConstants {
```

Comment:
Are these actually different to what we had before? If that's the case that's a breaking change. IMO we should introduce new tags next to the current ones and deprecate the old ones. 

Also how will ensure the compatibility between this class and semantic conventions of OpenTelemetry? Are we to be compatible with all the versions? Which version is this compatible with, for which date? Semantic Conventions are not stable and they can be changed at any point in time. What's our strategy for such breakages?

Reply:
However these Tags are potentially more used by Camel end users and if possible we could have the old tags deprecated and if possible let the old tags delegate to the new tags.